RPS_ver2.py

```diff
@@ -38,22 +44,25 @@ def decide_winner(p1, p2):
 while True:
 
     # gather the player choices and convert to lowercase for easy comparison later
-    player1 = raw_input('Player one.. Rock, Paper, or Scissors? ').lower()
-    player2 = raw_input('Player two.. Rock, Paper, or Scissors? ').lower()
+    player_one_input = raw_input(rps_string.format("One")).lower()
+    player_two_input = raw_input(rps_string.format("Two")).lower()
 
     # If the test_input() returns
-    if test_input(player1) == True and test_input(player2) == True:
-        break
-
-    else:
+    if player_one_input not in SET_PLAYS or player_two_input not in SET_PLAYS:
```

Comment:
In a game this simple, it actually DOESN'T make sense to worry about that test_input function.  If we were checking many different states, then it would be use it, but we're only checking two, and we're only using it in one place.